### PR TITLE
Welcome Page

### PR DESCRIPTION
**Pull request by spirographics**
Created a welcome / quick start page that only covers the most basic rules and how to create a first image for users less familiar with Discord.

### CODE REVIEW

File: WELCOME.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<br/>
+
+---
+--- 
+
+<br/>
+
+
+## Create your First Image 
+Once you have joined the Discord server you will be able to create a limited number of images before needing to subscribe.
```
Comment (gadgetsam) [inline WELCOME.md:22]: Maybe specify around 25 images

File: WELCOME.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+![Select a Newbies Channel](https://user-images.githubusercontent.com/105028755/167752981-596a4819-163b-4c4c-9241-adfd6231a1f4.jpg)
+
+<br/>
+
+
+#### 2. Use the /Imagine command
+
+To use the Midjourney Bot use a [Discord Slash Command](https://support.discord.com/hc/en-us/articles/1500000368501-Slash-Commands-FAQ) and type`/imagine`.
+Immediately after typing `/imagine` press spacebar, tab, or enter to create the **prompt** field.
+Slash Commands will only work in *Bot Channels*. Typing `/Imagine` will not work in `#Support`, `#Discussion,` and other general channels.
```
Comment (gadgetsam) [inline WELCOME.md:34]: newbies can't see discussion channel channel so I would remove the reference to it. Also maybe add something that if you have any questions go to #support channel

File: WELCOME.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Upscaling an image generates a larger roughly 1024x1024 pixel version of the selected image while adding additional details.
+
+![Midjourney Buttons](https://user-images.githubusercontent.com/105028755/167755032-fe6935a9-b6a2-4b80-8f73-13916c170ceb.png)
+
+<br/>
+
+#### 5. Save Your Image
+![MJ_Daisies](https://user-images.githubusercontent.com/105028755/167755159-875d58d0-12b0-4e8f-ac84-1cea75590fd8.png)
+
+Your upscaled image will immediately be visible on [gallery.midjourney.com](https://gallery.midjourney.com/)
```
Comment (gadgetsam) [inline WELCOME.md:62]: Both upscales and grids are available on the gallery :)

File: WELCOME.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+#### 6 Send your results to your DMs
```
Comment (gadgetsam) [inline WELCOME.md:64]: Also maybe add a 7th thing about how to make a thread

**Review by gadgetsam — CHANGES_REQUESTED** [review]
Looks really good! I just added a couple quick changes :)

**Comment by spirographics** [comment]
@gadgetsam changes made.

**Review by gadgetsam — APPROVED** [review]

**Pull request merged**